ci/prs.yml

```diff
@@ -170,3 +170,22 @@ jobs:
             tell_slack "<@${user}> <https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|Build $(Build.BuildId)> for <https://github.com/digital-asset/daml/pull/$(pr.num)|PR $(pr.num)> has completed with status $(build_status) (changelog: $(changelog_status))." \
                        "$(Slack.team-daml-ci)"
         fi
+
+- job: self_service_compat_test
+  pool:
+    vmImage: ubuntu-20.04
+  steps:
+  - checkout: self
+  - template: bash-lib.yml
+    parameters:
+      var_name: bash-lib
+  - bash: |
+      set -euo pipefail
+      source $(bash-lib)
+      COMMIT=$(git rev-parse HEAD^2)
```

Comment:
Out of curiosity, why `HEAD^2`?

Reply:
PR builds (so whatever's in `ci/prs.yml`) run on a synthesized merge commit between the branch and `main`, so `HEAD^1` is the main commit and `HEAD^2` is the branch commit (not to be confused with `HEAD~2`, which would be the parent of the main commit).